src/main/java/com/automation_boss/inventory/Chair.java

```diff
@@ -18,4 +19,21 @@ public void print(PrintStream out){
         out.println("Legs count - " + legs);
         out.println("Chair shape is - " + shape);
     }
+
+    @Override public boolean equals(Object o) {
```

Comment:
Please format code in this way
```java
@Override
public boolean equals(Object o) {
...
}
```

This is applicable for all methods with annotations (like `@Override`).